api/server/object.go

```diff
@@ -88,18 +86,14 @@ func (tapi ToDDApi) DeleteObject(w http.ResponseWriter, r *http.Request) {
 
 	deleteInfo := make(map[string]string)
 
-	body, err := ioutil.ReadAll(r.Body)
+	err := json.NewDecoder(r.Body).Decode(&deleteInfo)
```

Comment:
As we discussed in the email thread, I'm less familiar with the NewDecoder approach. How will this line as written handle an error in NewDecoder? Would it be worth breaking this up into multiple lines, so we can handle errors appropriately? Or is it moot, since err in this case will be set regardless of which function has an error? Just thinking out loud, really.